src/book/02-system/01-nubots/images/power_plant.svg

```diff
@@ -0,0 +1,721 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This SVG file can be optimized or a smaller size using [SVGOMG](https://jakearchibald.github.io/svgomg/).

Would also be nice, but not necessary, if it had a transparent background.